pkg/controllers/framework/interface.go

```diff
@@ -37,6 +38,9 @@ type ControllerOption struct {
 	InheritOwnerAnnotations bool
 	WorkerThreadsForPG      uint32
 	WorkerThreadsForGC      uint32
+
+	// Config holds the common attributes that can be passed to a Kubernetes client on initialization.
```

Comment:
Kubernetes client on initialization. =》Kubernetes client and used by controllers registered by the user.